contracts/voting-verifier/src/state.rs

```diff
@@ -60,6 +60,18 @@ impl PollMessage {
     }
 }
 
+#[cw_serde]
+pub struct PollWorkerSet {
+    pub operators: Operators, // operators are stored for migration purposes in case the hash changes
+    pub poll_id: PollID,
+}
```

Comment:
Can we merge this with `PollMessage` to something like 
```
pub struct PollContent<T> {
    pub content: T<
    pub poll_id: PollId,
    pub index_in_poll: usize
}
```
that means we need to deal with fewer structs, both have the same structure and we don't need to put the explicit `0` in `Ok(verification_status(deps, stored.poll_id, 0))`, which is a bug waiting to happen, if WorkerSet can ever be confirmed together with other poll items.

As a sidenote, could you rename `PollID` to `PollId`? The rust naming conventions are quite clear that acronyms should also use camel case

Reply:
done. Since we have a common structure now I also tried to remove some other duplicated code inside the verification status functions, take a look at this commit: https://github.com/axelarnetwork/axelar-amplifier/pull/201/commits/0123282503ab2b407f4bc00c35d465f52ee8a196